modules/system/classes/VersionManager.php

```diff
@@ -544,6 +548,54 @@ protected function extractScriptsAndComments($details)
             $scripts = [];
         }
 
-        return array($comments, $scripts);
+        return [$comments, $scripts];
+    }
+
+    /**
+     * Get the current version plugin
+     *
+     * @param $plugin
+     * @return int
+     */
+    public function getCurrentVersion($plugin)
```

Comment:
Return type should be a string.
```suggestion
    public function getCurrentVersion($plugin): string
```